produce world map of AA countries

joseepoirier opened the pull request:
Manually created the entry for Malawi to simplify the graph. We'll need to adapt the code when we have > 1 dual shock context. 
I'd vote for continuing to distinguish between dry spells and drought, or we combine them and label the category drought/dry spells since they are distinct shocks.

Tinkaa commented on utils_general/aa-countries.R:
```diff
@@ -0,0 +1,75 @@
+###
```
Not regarding this line, but the directory structure. I would propose to not place this in utils_general as this are really pipeline scripts. Instead I would propose to make a `analyses/glb` folder, and place it there. What do you think?

joseepoirier replied:
Oh I hadn't noticed glb. Much better location for it.

joseepoirier replied:
I need to figure out how to move the Rproj file with it..

Tinkaa replied:
`glb` doesn't exist yet😁 but think this is a perfect occasion to create it

Tinkaa commented on utils_general/aa-countries.R:
```diff
@@ -0,0 +1,75 @@
+#library(rvest)
+#library(magrittr)
+#library(ggmap)
+#library(stringr)
```
remove unused libraries

joseepoirier replied:
Ha! forgot to do that. Thanks!

Tinkaa commented on utils_general/aa-countries.R:
```diff
@@ -0,0 +1,75 @@
+# list countries and shocks
+drought <- c("Somalia", "Ethiopia", "Niger", "Burkina Faso")
+#dry_spells <- c("Malawi")
+#floods <- c("Bangladesh", "Nepal", "Malawi", "Chad", "South Sudan")
```
remove commented lines

joseepoirier replied:
I was leaving them in so we can easily update it / change how we represent MWI. I guess if the overall structure & pattern are clear I can remove them.

Tinkaa replied:
to me the overall structure is clear

Tinkaa commented on utils_general/aa-countries.R:
```diff
@@ -0,0 +1,75 @@
+        ,axis.title = element_blank()
+        ,axis.ticks = element_blank()
+        ,panel.background = element_rect(fill = "white")
+        #,plot.background = element_rect(fill = "#444444")
```
here and above, remove commented lines

Tinkaa commented on utils_general/aa-countries.R:
```diff
@@ -0,0 +1,75 @@
+        #,plot.background = element_rect(fill = "#444444")
+        ,panel.grid = element_blank()
+  )
+
```
don't see code to save the png? (do comment out that line but good to keep it somewhere)

joseepoirier replied:
The saving function distorts the image when it saves it. I've already spent a few hours trying to figure it out. Would say to leave it as, keeping the manually saved png for now, and I'll look into the saving issue later.

Tinkaa reviewed (commented):
Beautiful how ggplot can so easily create nice plots, and you nicely documented the code! 

Few things
- as commented would place the files in a different directory
- I am not a big fan of putting the pngs in the repo. Would it be alright to put it on the GDrive (`processed/glb/plots` for example) or is there a specific reason you saved it here? 
- I think it would be really nice to add the map to the repo main readme. What do you say?

joseepoirier commented:
@Tinkaa I addressed your helpful comments, thanks. Can you take a quick look so we can merge?

Tinkaa commented:
@joseepoirier Looks great! The only thing is that when I open the png now saved in `processed/glb/plots`, it doesn't look good, it is very stretched. So maybe change how you save the png? But regardless of that, ready to be merged imo.

Tinkaa commented:
Aah and the png seems to still be saved on the github, would you agree to remove it from there?

joseepoirier commented:
> Aah and the png seems to still be saved on the github, would you agree to remove it from there?

Ugh that's me forgetting to actually removing it after realising the saving function issue. Sorry - I deleted it and also moved the R file + Rproj to `analyses/glb`
